README.md

```diff
@@ -1,53 +1,74 @@
-# semgrep action
+# Semgrep Action
 
-This action runs [semgrep](https://semgrep.dev) and returns the output
+[![r2c community slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=for-the-badge&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
 
-## Inputs
+This GitHub Action reviews pull requests with [Semgrep](https://github.com/returntocorp/semgrep)
+whenever a new commit is added to them.
+It reports as failed if there are any new bugs
+that first appeared in that pull request.
 
-### `config`
+## Usage
 
-The config `file|directory|yaml_url|tar|url|registry_name`.
+To start checking all pull requests,
+add the following file at `.github/workflows/semgrep.yml`:
 
-### `output`
+```yaml
+name: Semgrep
+on: [pull_request]
+jobs:
+  semgrep:
+    runs-on: ubuntu-latest
+    name: Check
+    steps:
+    - uses: actions/checkout@v1
+    - name: Semgrep
+      id: semgrep
+      uses: returntocorp/semgrep-action@v1
+      with:
+        config: r/r2c
+```
 
-The output arg `file|url`
+Note that the `r/r2c` config value
+will enable all 160+ checks from [our registry](https://semgrep.live/r).
```

Comment:
The 160+ checks will probably be out of date soon. Want to just be non-specific here and remove the number?

Reply:
@underyx can you also update to use `https://semgrep.live/c/r/all` I'm moving away from `r2c.` prefix as this is our community registry after a good idea from @dlukeomalley

Reply:
note that `r2c.` and `/c/r/r2c` sill work and get remapped server side

Reply:
@pabloest I wanted to hint at the order of magnitude though, so people know what they're getting themselves into :D

How about "all the hundreds of checks"? Is that okay or still nah?

Reply:
@underyx Yep I think that works.